eventually/amazons3/views.py

```diff
@@ -65,9 +66,9 @@ def post(self, request):
                                          Body=image_for_upload.read(),
                                          ContentType=mime_type,
                                          Key=image_key)
-            return JsonResponse({"image_key": response.key}, status=200)
+            return JsonResponse({"image_key": response.key}, RESPONSE_200_OK)
```

Comment:
does this work correctly?, because `RESPONSE_200` is the http response object. does it not break anything when we add it to `JsonResponse` object?

Reply:
You're correct, missed this fact. Fixed.